### DISCUSSION TITLE
OPA / OPA Gatekeeper Logging Suggestions

### DISCUSSION THREAD

**Discussion by scoobersdad** [open-policy-agent/community#189] [Gatekeeper]
I am running OPA Gatekeeper 3.7.1 and OPA 0.40.0-debug on Openshift. I use helm to deploy tomcat charts and validate labels. Everything is working as excepted, if the chart has a route definition the webhook fails. The chart will deploy fine without the route defined. I have enable debug logs in OPA but so far I can not see why its failing. The webhook failure:

Error: UPGRADE FAILED: failed to create resource: Internal error occurred: admission plugin "ValidatingAdmissionWebhook" failed to complete validation in 13s

Trying to find any other logging suggestion to help resolve the issue. 

Thanks in advance

**Comment by sozercan** [comment]
@scoobersdad can you provide more information? Where is ValidatingAdmissionWebhook failure coming from? Is that from Gatekeeper or Tomcat? 

Are there any policies in the cluster? Please provide more information.
